apps/astarte_housekeeping/test/test_helper.exs

```diff
@@ -1,3 +1,5 @@
 Mimic.copy(Xandra)
 Mimic.copy(Xandra.Cluster)
 ExUnit.start(capture_log: true)
+Mimic.copy(Astarte.Housekeeping.Queries)
+ExUnit.start()
```

Comment:
Repeated `ExUnit.start()` from line 3, let's keep the one with `capture_log: true`